fw/http.c

```diff
@@ -1655,13 +1655,15 @@ tfw_http_mark_wl_new_msg(TfwConn *conn, TfwHttpMsg *msg,
 	if (!tfw_wl_marks.mrks || !(TFW_CONN_TYPE(conn) & Conn_Clnt))
 		return;
 
-	if (bsearch(&skb->mark, tfw_wl_marks.mrks, tfw_wl_marks.sz,
-		    sizeof(tfw_wl_marks.mrks[0]), tfw_http_marks_cmp))
-	{
+	if (tfw_http_mark_is_in_whitlist(skb->mark))
 		__set_bit(TFW_HTTP_B_WHITELIST, msg->flags);
-	}
 }
 
+bool tfw_http_mark_is_in_whitlist(unsigned int mark)
+{
+	return !!bsearch(&mark, tfw_wl_marks.mrks, tfw_wl_marks.sz,
+			 sizeof(tfw_wl_marks.mrks[0]), tfw_http_marks_cmp);
+}
```

Comment:
We (and the Linux kernel) follow the common C practice to place a callee above the caller and we place function return type on a separate line

Reply:
yes fixed